### ISSUE TITLE
PIT will not consider exclude_names and exclude_types 

### ISSUE THREAD

**Issue by David-Nevezi-Strango** [eml-eda/plinio#11]
Torch version 2.3.0 with torchvision 0.18.0, both with cu118 from torch.

I have tried converting the model from this notebook to a PIT model, has problems at the last activation layer ( `out = F.sigmoid(out)`).
Throws error: Unsupported node: sigmoid, with n.op being "callback" and n.target being "sigmoid".

Tried changing class structure, utilizing torch.nn.Sigmoid. no change

notebook: https://www.kaggle.com/code/boascent/multi-label-image-classification-pytorch-gpu/notebook

**Comment by danielepagliari** [comment]
Hi David,

Again, sorry for the huge delay. I think this doesn't have to do with exclude_names and exclude_types. Those two parameters only specify the layers that are not converted to equivalent PIT modules. Even excluded layers will throw exceptions if PIT cannot parse them correctly to understand the structure of the NN graph (which is necessary for computing cost).

There are two options:
- Call the sigmoid function (if needed at all and can't be bypassed by using a loss function that takes logins directly) outside of the nn.Module you are converting to PIT.
- Adding support for F.sigmoid() in PIT. For this, if you plan to submit a PR, I'll be happy to review it.

Best.

Daniele
